services/skipchain/skipchain_test.go

```diff
@@ -16,7 +16,8 @@ import (
 )
 
 func TestMain(m *testing.M) {
-	log.Info("Skipping all skipchain tests for now as it randomly fails on master. https://github.com/dedis/cothority/pull/483")
+	//j	log.Info("Skipping all skipchain tests for now as it randomly fails on master. https://github.com/dedis/cothority/pull/483")
```

Comment:
I **hope** it works, we're not sure since the bug does not occur deterministically...